core/src/sigverify_stage.rs

```diff
@@ -188,11 +216,14 @@ impl SigVerifyStage {
                 batches[batch_index].packets[packet_index]
                     .meta
                     .set_discard(true);
+                fail += 1;
```

Comment:
I don't think we need to count here. We know the number of packets entering the function already. Just subtract with the number of packets you tell it to give you back.

Reply:
I don't get it.  where do we drop the packets?